### DISCUSSION TITLE
Morgen bellen?

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#21] [Ideas]
@rosa33783 stelde in WhatsApp voor om morgen te bellen en dacht zet het even hier. Hoe laat kan iedereen?

**Comment by rosa33783** [comment]
Ik kan om 11.00

**Reply by D0nnaz** [reply]
@dinhs-frontend @Sensinki 11 uur doen?

**Reply by Sensinki** [reply]
Ik slaap nog steeds niet 🥲, maar ik zal proberen om om 11 uur wakker te zijn 👀
